packages/studio/web/src/Pages/Explore/TraceList.tsx

```diff
@@ -134,10 +136,8 @@ export default function TraceList() {
     },
   })
 
-  const sortedTraces = data?.rows?.sort((a: any, b: any) => {
-    const bigIntMin = (...args: bigint[]) =>
-      args.reduce((m, e) => (e < m ? e : m))
-
+  const traces: any[] = data?.rows !== undefined ? [...data.rows] : []
```

Comment:
Sounds good, but let's see later if we can do sorting in the service/view/queries perhaps? Maybe a sort option?

Reply:
Yeah agreed